exporters/metric/cortex/cortex.go

```diff
@@ -185,7 +186,7 @@ func (e *Exporter) ConvertToTimeSeries(checkpointSet export.CheckpointSet) ([]*p
 func createTimeSeries(record metric.Record, value apimetric.Number, extraLabels ...string) *prompb.TimeSeries {
 	sample := prompb.Sample{
 		Value:     value.CoerceToFloat64(record.Descriptor().NumberKind()),
-		Timestamp: record.EndTime().Unix(),
+		Timestamp: record.EndTime().UnixNano() / int64(time.Millisecond),
```

Comment:
To avoid a Y2K-esq bug and help my physics based brain can we change these to explicitly include the base time unit we are converting from? Almost certain the compile will optimize it away, but it is useful for the reader.

```suggestion
		Timestamp: int64(time.Nanosecond) * record.EndTime().UnixNano() / int64(time.Millisecond),
```

Reply:
Definitely see how that's clearer!